### ISSUE TITLE
Amalgamated build

### ISSUE THREAD

**Issue by DaulPavid** [bkaradzic/bimg#3]
Unlike bx and bgfx, there's no amalgamated build for this project. This makes it hard to integrate new bgfx versions into my build system since bgfx now includes bimg.

**Comment by ppiecuch** [comment]
I have prepared amalagamted version of bimg (for the same reasons you mentioned) but I have no fork of bimg (and this required quite a few changes): my working copy of the original repository is here: bimg-16-04-2017.zip <https://github.com/ppiecuch/bgfx/blob/master/qt/bx-qt/bimg-16-04-2017.zip> if you are interested.

Btw: would it be possible to rename amalgamated files to eg: bgfx-amalgamated, bx-amalgamted (I named bimg: bimg-amalgamated). Sadly qmake has problems working with files with same name.

Regards
Pawel
